armel/build.sh

```diff
@@ -1,20 +1,33 @@
 #!/bin/bash
 set -e
 
+# comparing version: http://stackoverflow.com/questions/16989598/bash-comparing-version-numbers
+function version_le() { test "$(echo "$@" | tr " " "\n" | sort -V | tail -n 1)" == "$1"; }
+
 # set env var
 NODE_VERSION=$1
 ARCH=arm
 ARCH_VERSION=armel
-TAR_FILE=node-v$NODE_VERSION-linux-$ARCH_VERSION.tar.gz
+TAR_FILE=node-v$NODE_VERSION-linux-$ARCH_VERSION
 BUCKET_NAME=$BUCKET_NAME
 
+# --with-arm-fpu flag is not available for node versions 0.12.x and 0.10.x
+if version_le "$NODE_VERSION" "4"; then
+	BUILD_FLAGs='--with-arm-float-abi=softfp --with-arm-fpu=vfpv3'
```

Comment:
@petrosagg, I took this from node configure https://github.com/nodejs/node/blob/v0.12.9/configure#L510 but apparently. NodeJS dropped support armv5 long time ago so this setting is not for armv5 :/ Changed to `vfp`